.github/workflows/cldf-validation.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.9]
+        python-version: [3.9, "3.10", 3.11, 3.12]
```

Comment:
You can remove 3.9

Reply:
CLDF validation does not really require being run with different Python versions. So I'd just stick to `"3.10"`.

Reply:
Thanks for your comments. I adopted this.